gensim/matutils.py

```diff
@@ -1328,10 +1328,23 @@ class MmReader(object):
     This allows us to process corpora which are larger than the available RAM.
 
     """
+    @contextmanager
```

Comment:
Please move this function to `utils.py`  (this isn't related with MmReader, this can be useful in other cases)